app/views/users/index.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1> Here is list of users and their corresponding number of posts
-</h1>
+<div class="users-index-wrapper">
+<h1 class="header-text">Here is list of users and their corresponding number of posts</h1>
+
+<ul class="users-container">
+  <% @users.each do |user| %>
```

Comment:
- [ ] [optional] It would be nice to use `app/views/users/_user.html.erb`, below as a better practice instead of using `@objects.each` in rendering the users.